rclpy/test/test_create_node.py

```diff
@@ -82,6 +83,29 @@ def test_create_node_with_parameter_overrides(self):
             ]
         ).destroy_node()
 
+    def test_create_node_disable_rosout(self):
+        node_name = 'create_node_test_disable_rosout'
+        namespace = '/ns'
+        node = rclpy.create_node(
+            node_name, namespace=namespace, context=self.context, enable_rosout=False)
+        node.destroy_node()
+
+    def test_create_node_rosout_qos_profile(self):
+        node_name = 'create_node_test_rosout_rosout_qos_profile'
+        namespace = '/ns'
+        node = rclpy.create_node(
+            node_name, namespace=namespace, context=self.context, enable_rosout=True,
+            rosout_qos_profile=qos_profile_sensor_data)
+        node.destroy_node()
+
+    def test_create_node_rosout_qos_depth(self):
+        node_name = 'create_node_test_rosout_rosout_qos_depth'
+        namespace = '/ns'
+        node = rclpy.create_node(
+            node_name, namespace=namespace, context=self.context, enable_rosout=True,
+            rosout_qos_profile=10)
+        node.destroy_node()
```

Comment:
I'm not a huge fan of tests that don't `assert` anything.  I guess these do test that this isn't throwing an exception, but they aren't testing that what we asked for actually happened.  Could we assert that `rosout` exists (or doesn't exist), and that it has the QoS profile we provided?

Reply:
addressed in https://github.com/ros2/rclpy/pull/1376/commits/987bc366b2c2173925a95f4bed342cf91f25b9f7